util/settings/settings.go

```diff
@@ -2223,3 +2229,12 @@ func (mgr *SettingsManager) GetResourceCustomLabels() ([]string, error) {
 	}
 	return []string{}, nil
 }
+
+// GetIsImpersonationEnabled returns true if application sync with impersonation feature is enabled in argocd-cm configmap
+func (mgr *SettingsManager) GetIsImpersonationEnabled() bool {
```

Comment:
would it be better to just name it  `IsImpersonationEnabled`  ?

Reply:
I was following the pattern used for other boolean variables which all had a `Get` prefix. I beleive it is not required to have that prefix. Changed the method name to `IsImpersonationEnabled` as suggested. Thanks @akram.